supertokens_python/recipe/session/with_jwt/session_class.py

```diff
@@ -93,5 +93,7 @@ async def update_access_token_payload(
             new_access_token_payload, user_context
         )
 
+    # TODO: Add missing override functions
```

Comment:
TODO

Reply:
Only `update_access_token_payload` is overriden which is already done. So this comment wasn't required.

Done.